### PR TITLE
🐛 destination snowflake v2: uppercase database name

### PR DESCRIPTION
**Pull request by edgao**

### CODE REVIEW

**Review by evantahler — APPROVED** [review]
😠 ❄️

File: docs/integrations/destinations/snowflake.md [hunk 1]
```diff
@@ -271,6 +271,7 @@ Otherwise, make sure to grant the role the required permissions in the desired n
 
 | Version         | Date       | Pull Request                                               | Subject                                                                                                                                                         |
 |:----------------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2.1.6           | 2023-08-28 | [\#29878](https://github.com/airbytehq/airbyte/pull/29878) | Destinations V2: Fix existing detection of existing table                                                                                                       |
```
Comment (evantahler) [inline docs/integrations/destinations/snowflake.md:274]:
```suggestion
| 2.1.6           | 2023-08-28 | [\#29878](https://github.com/airbytehq/airbyte/pull/29878) | Destinations V2: Fix detection of existing table by ensuring upper-case DB names                                                                                                     |
```

**Comment by edgao** [comment]
manually cancelled the CI run to reduce inbox noise

**Comment by octavia-approvington** [comment]
This is really good <br/> simply the best

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**